.travis/install.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
```

Comment:
Could you add `set -e` to `linux..before_install.sh`, `linux.win32.before_install.sh`, `linux.win32.install.sh`, etc.?